testsuite/features/step_definitions/api_common.rb

```diff
@@ -8,6 +8,38 @@
 
 # system namespace
 
+When(/^I delete all the imported terminals$/) do
```

Comment:
I wonder if this implementation will fit better in `features/step_definitions/retail_steps.rb`

Reply:
Mmm, it also only use API. I don't have strong opinion

Reply:
I was already merged like this in 5.1. I will keep it as is to avoid discrepancy.